package.json

```diff
@@ -25,6 +26,7 @@
   "homepage": "https://github.com/Interencheres/cpm-config#readme",
   "devDependencies": {
     "eslint": "^5.12.0",
+    "handlebars": "^4.5.0",
```

Comment:
Je comprends pas pourquoi l'ajout de handlebars

Reply:
il y avait des warning cf:
```
Handlebars: Access has been denied to resolve the property "statements" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
Handlebars: Access has been denied to resolve the property "branches" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
Handlebars: Access has been denied to resolve the property "functions" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
Handlebars: Access has been denied to resolve the property "lines" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
```
en forcant une version plus recente on les fait disparaitre

Reply:
de ce que je comprends, c'est une dépendance de jest, est-ce que c'est pas plutôt jest qu'il faut mettre à jour ?

Reply:
effectivement, j'avais run le npm update mais on restait bloquer sur la derniere version 22, en forcant la derniere 26 le warning disparait, 25e75af08398de43c56375a91b89ab8bc0641094

Reply:
La mise à jour d'eslint casse le lint par contre 😕

Reply:
yep: 9f9f5a25fa99f013565b99f673a91cadd4b04dbe